Login.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
The filename should always be lowercase since certain servers treat different casing as different files and it may cause `404` mistakes.

Reply:
Will change it